src/hooks/useFetch.ts

```diff
@@ -67,7 +67,7 @@ const useFetch = <Data = any, FetchParams extends any[] = any[]>(
         setStatus('err');
       }
     },
-    [isReady],
+    [fetchData, isReady],
```

Comment:
This will redefine `getData` and `refresh` at each render, since we are also using arrow functions `useFetch(() => astarte.client.fetchStuff())`.
We will need to keep an eye on using arrow functions in `useFetch` from now on, to avoid unexpected behavior/rerenders